plugin/vcs/github/github_test.go

```diff
@@ -91,6 +91,71 @@ func TestProvider_FetchUserInfo(t *testing.T) {
 	assert.Equal(t, want, got)
 }
 
+func TestProvider_FetchCommitByID(t *testing.T) {
```

Comment:
```suggestion
func TestProviderFetchCommitByID(t *testing.T) {
```
let's keep it consistent with other test name

Reply:
This naming style is consistent in all other tests in `github_test.go` and `gitlab_test.go`. I think this is not concern within PR, but we can discuss and address this in a separate PR.

Reply:
SG. In general, sub-tests use the name of TestProvider_FetchCommitByID.

Reply:
I checked all [existing 82 test functions](https://sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/bytebase/bytebase%24+func+Test&patternType=literal), it seems that only I have written tests for struct methods, which happens to use the pattern `Test<T>_<method>` ... that means unless we want to eliminate the use of underscore `_` in tests name, there is no change needed at this point.